core/experiments/manager.php

```diff
@@ -382,6 +382,14 @@ private function add_default_features() {
 			'generator_tag' => true,
 			'default' => self::STATE_ACTIVE,
 		] );
+
```

Comment:
@hein-obox - maybe place it in the dedicated module 
Also, does it have a tag ?

Reply:
@TzviRabinovitch 
We don't have any files in the Core repo for the Indication Popover. Would you like me to create an empty module for this?

I don't think that a generator tag is relevant in this content.

Wdyt?